README.md

```diff
@@ -64,6 +64,10 @@ docker-compose -d up postgres
 ```
 dotnet ef migrations add <Migration-name> --project DeUrgenta.Domain --startup-project DeUrgenta.Api
 ```
+### Adding EF Core migration to User.Api
```

Comment:
Eu aici in readme as pune si cum sa rulam database update 
```
dotnet ef database update --startup-project ..\DeUrgenta.Api --context UserDbContext
```

Reply:
@trupci do we need this ? because we call Migrate database when adding db context